### PR TITLE
ML cleanup

### PR DESCRIPTION
**Pull request by ndan029**
Remove shell command from ml
Add proxy container to lite mode
rearrange CA key generation and docker image building
remove deep-clean feature, flag is still there as no-op 
pin requirements for docs image
remove import/export subcommand from ml manage command

### CODE REVIEW

File: ml [hunk 1]
```diff
@@ -103,59 +103,24 @@ def init(
     quiet=False,
     nomaps=False,
     lite=False,
+    deep_clean=False,
```
Comment (hohumsup) [inline ml]: is there a reason why we are keeping deep_clean here if it is deprecated?

File: ml [hunk 2]
```diff
@@ -204,106 +199,95 @@ def init(
...
-
-        if make_migrations:
-            MAKE_MIGRATIONS_FLAG = "true"
-        else:
-            MAKE_MIGRATIONS_FLAG = "false"
-
-        # Don't run map tile server if there are no tile sets available
-        available_mbtiles = glob.glob(MAP_TILES_PATH + "*.mbtiles")
+    # Don't run portainer if "lite" flag is set
+    if lite:
```
Comment (supr3m3) [inline ml:222]:
```suggestion
     if lite:
        SERVICES.discard("portainer")
        SERVICES.discard("docs")
        SERVICES.discard("event_generator")
        SERVICES.discard("pulsar")
```

Comment (hohumsup) [inline ml:221]: think we can remove zookeepers, bookkeepers, and pulsar proxy as these services don't exist anymore

File: ml (hunk 1 above)
Comment (ndan029) [reply ml]: mainly for backwards compatibility, but I could see an argument to take it out completely

**Pull request closed without merging**